README.md

```diff
@@ -1,5 +1,10 @@
 # eQ Census Survey Runner v3
-[![Build Status](https://travis-ci.org/ONSdigital/eq-survey-runner.svg?branch=master)](https://travis-ci.org/ONSdigital/eq-survey-runner) [![codecov](https://codecov.io/gh/ONSdigital/eq-survey-runner/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/eq-survey-runner) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/82e63fc5bc5c43e8ba1ba6d13bfb4243)](https://www.codacy.com/app/ONSDigital/eq-survey-runner)
+[![Build Status](https://travis-ci.org/ONSdigital/eq-survey-runner.svg?branch=v3)](https://travis-ci.org/ONSdigital/eq-survey-runner) 
+[![codecov](https://codecov.io/gh/ONSdigital/eq-survey-runner/branch/v3/graph/badge.svg)](https://codecov.io/gh/ONSdigital/eq-survey-runner/branch/v3) 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a2bc191a96e546b99a6d1c33b9b0ed62)](https://www.codacy.com/app/MebinAbraham/eq-survey-runner?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ONSdigital/eq-survey-runner&amp;utm_campaign=Badge_Grade)
+
+This version of runner looks at optimising the product to accommodate the Census. 
+This involves changes to some of the fundamentals of how [runner currently works](https://github.com/ONSdigital/eq-survey-runner). This includes breaking changes and new architecture design changes.
```

Comment:
I think we need to expand on the reasoning for having a v3. Something like:

We need make some major changes to survey runner to develop the features required for the Census. These changes are a combination of fixing technical debt we've built up and adding new features. While these changes would be possible to implement in v2, having to develop them in such a way to be compatible with existing data and rolling deploys will add an un-acceptable overhead. The changes we're aware of now:

- architecture changes: 
  - block variants for proxy or past/present versions of content
  - piping in structured schema rather than inlined jinja filters
  - routing refactor for optimum performance
  - redesign how repeating groups and association with driving questions works
- new features:
  - who lives here (collecting householders)
  - relationships
  - hub and spoke navigation

Existing v2 schemas will be easily migrate-able to the new format when v3 is stable. This will be clearly documented and there is the possibility of scripts being provided to migrate schemas. Once we know more about the implementation a decision will be made whether answer store migrations will be written for compatibility with existing data.

Reply:
🙇